docs/docs/tutorial/index.md

```diff
@@ -121,9 +123,8 @@ The Hydra Head protocol requires a connection to Cardano layer 1 to verify and p
 
 Download the latest blockchain snapshot using `mithril-client` configured for the `preprod` network:
 
-
 ```shell
-mithril-client cardano-db download latest
+mithril-client --origin-tag HYDRA cardano-db download latest --include-ancillary
```

Comment:
What is this `--origin-tag` doing? The help page here did not really answer it: https://mithril.network/doc/manual/develop/nodes/mithril-client

Reply:
That's a way to isolate traffic coming from partners, CI, explorer and benchmarks. 
We use these origins to filter metrics in our aggregator dashboards.